marketplaceItems/sitPoint/README.md

```diff
@@ -0,0 +1,54 @@
+# Sit Point Script
+
+## Description
+
+Make any entity sittable whether it's a chair, bench, couch, log, or even a tree stump.
+
+Either import Sit Point from the marketplace to put on entities for one or more seats per entity or make a sittable entity yourself with one seat! See **Setup** > **Add scripts to your entity**
+
+## Releases
```

Comment:
this section will have to be updated after the PR is approved (not a problem, just making a note)